.github/workflows/release-spruce.yaml

```diff
@@ -0,0 +1,53 @@
+name: 'Test PyPI Release: Spruce build (pinecone-client-spruce)'
+
+on:
+  workflow_dispatch: {}
+
+jobs:
+  testing:
+    uses: './.github/workflows/testing.yaml'
+
+  version-and-release-spruce:
```

Comment:
Initially I wanted to leverage `publish-to-pypi` which the regular `release` workflow calls: https://github.com/pinecone-io/pinecone-python-client/blob/main/.github/workflows/publish-to-pypi.yaml

However, since we're not actually bumping the version for Spruce, it felt odd to make a lot of changes to support that. Instead I opted to use the same flow for publishing that we leverage in `nightly-release`.